Sources/NetworkProtection/NetworkProtectionDeviceManager.swift

```diff
@@ -59,12 +63,12 @@ public actor NetworkProtectionDeviceManager: NetworkProtectionDeviceManagement {
 
     private let isSubscriptionEnabled: Bool
 
-    public init(environment: VPNSettings.SelectedEnvironment,
+    public init(settings: VPNSettings,
```

Comment:
As mentioned above this should be rolled back - we can't support updating the environment without restarting the connection.